src/sdk/pynni/tests/test_compression_utils.py

```diff
@@ -11,13 +11,13 @@
 
 from nni.compression.torch import L1FilterPruner
```

Comment:
Would you please add test cases to verify the model speedup correctness for resnet, squeezenet and mobilenet like this test case?
https://github.com/microsoft/nni/blob/master/src/sdk/pynni/tests/test_model_speedup.py#L106

Reply:
Sure~